### ISSUE TITLE
Conda install of harp with Python 3.7.6 fails

### ISSUE THREAD

**Issue by calekochenour** [stcorp/harp#225]
I'm trying to install harp into a Conda environment, running Python 3.7.6. I have the correct channels, but Conda fails to solve the environment during install. Does harp support Python 3.7.x?

**Comment by svniemeijer** [comment]
Python 3.7 is supported.
Please explain exactly what you did and the error message you got.
Otherwise this issue will be closed

**Comment by calekochenour** [comment]
I first added the stcorp channel to my list of channels. With an active Conda environment, I ran the `conda install -c stcorp harp` and `conda install harp` separately with my terminal, but neither worked. The following error occurred each time. The environment was never able to solve.

<img width="944" alt="conda-install-harp" src="https://user-images.githubusercontent.com/54423680/86138095-39cafe00-baab-11ea-83b7-f5c79d9abbe4.PNG">

However, I was just successful installing it by creating a new Conda environment from an environment.yml file and `harp` within the list of dependencies. I should be able to proceed at this point. Though I am not sure why it wouldn't install from the terminal directly.
